huxley/www/static/js/huxley/components/DelegateSelect.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+ * Use of this source code is governed by a BSD License (see LICENSE).
+ */
+
+'use strict';
+
+var React = require('react');
+
+var DelegateSelect = React.createClass({
+  propTypes: {
+    onChange: React.PropTypes.func,
+    delegates: React.PropTypes.array,
+    selectedDelegateId: React.PropTypes.number
+  },
+
+  render: function() {
+    return (
+      <select
+        onChange={this.props.onChange}
+        value={this.props.selectedDelegateId}>
+        <option value="0">None</option>
+        {this.renderDelegateOptions()}
+      </select>
+    );
+  },
+
+  renderDelegateOptions: function() {
+    return this.props.delegates.map(function(delegate) {
```

Comment:
Little easier to read with an arrow function:
```js
return this.props.delegates.map((delegate) =>
  <option ...>
    {delegate.name}
  </option>
);
```
Also, this function doesn't even use `this`, so we didn't need to bind in the first place ;)